helm/portieris/templates/webhooks.yaml

```
@@ -1,12 +1,40 @@
+# an inoperative webhook, establishes the object in the helm manifest such that it is uninstalled on chart delete
 apiVersion: admissionregistration.k8s.io/v1beta1
 kind: MutatingWebhookConfiguration
 metadata:
   name: image-admission-config
+  annotations:
   {{ if .Values.UseCertManager }}
+    cert-manager.io/inject-ca-from: {{ .Release.Namespace }}/portieris-certs
+  {{ end }}
+  labels:
+    app: {{ template "portieris.name" . }}
+    chart: {{ template "portieris.chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+webhooks:
+  - name: trust.hooks.securityenforcement.admission.cloud.ibm.com
+    clientConfig:
+      service:
+        name: {{ template "portieris.name" . }}
+        namespace: {{ .Release.Namespace }}
+        path: "/admit"
+      {{ if not .Values.UseCertManager }}
+      caBundle: {{ .Files.Get "certs/ca.crt" | b64enc }}
+      {{ end }}
+    rules:
```

Looks good - but this needs a list to remain valid yaml

```suggestion
    rules: []
```

---

interesting, I did install it and it worked ! maybe the inop one doesnt matter